fiorella/src/sections/product.liquid

```diff
@@ -131,3 +124,11 @@
     ]
   }
 {% endschema %}
+
+<style type="text/css">
+  #Quantity {
+    background: white url(https://image.flaticon.com/icons/svg/156/156320.svg) right no-repeat;
```

Comment:
I suggest to download this asset because if that image is removed then the icon is not going to work anymore

Reply:
You're right. :+1: